content/posts/networkx/vf2pp/ISO-feasibility-candidates/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "ISO Feasibility & Candidates"
+date: 2022-07-11
+draft: false
+description: "
+Information about my progress on two important features of the algorithm.
+"
+tags: ["gsoc", "networkx"]
+displayInList: true
+author: ["Konstantinos Petridis"]
+
+---
+
+The previous post can be found [here]({{< relref "../node-ordering-Ti-updating" >}}), be sure to check it out, so you
+can
+follow the process step by step. Since then, another two very significant features of the algorithm have been
+implemented and tested.
+
+## Introduction
+
+As it was previously mentioned, in the ISO problem we are basically trying to create a **mapping** such that, every node
+from the first graph is matched to a node from the second graph. This searching for "feasible pairs" can be visualized
+by a tree, where each node is the candidate pair that we should examine. This can become much clearer if we take a look
+at the below figure.
+
+<center><img src="dfs.png" alt="DFS VF2++ example$"/></center>
+
+In order to check if the graphs $G_1$, $G_2$ are isomorphic, we check every candidate pair of nodes and if it is
+feasible, we extend the mapping and go deeper into the tree of pairs. If it's not feasible, we climb up and follow a
+different branch, until the mapping is extended to its full. In our example, we start by examining node 0 from G1, with
+node 0 of G2. After some checks that are skipped for simplicity (but can be perceived by observation), we decide that the
```

Comment:
For the sake of brevity, we could instead implore the reader to keep reading for details!

```suggestion
node 0 of G2. After some checks (details below), we decide that the
```